views/home.tmpl

```diff
@@ -245,7 +246,9 @@
                                 </span>
                             </div>
                             <div class="fs12 lh1rem text-black-50">
-                                <span data-target="homepage.targetPct">{{printf "%.2f" .PoolInfo.PercentTarget}}</span> % of target&nbsp;<span>{{intComma .PoolInfo.Target}}</span>
+                                <span data-target="homepage.targetPct">{{printf "%.2f" $percentchange}}</span>
+                                % {{if lt $percentchange 0.0}} under {{else}} over {{end}} a target of&nbsp;
```

Comment:
The space between the percent sign and the number is unusual.  Instead of `0.5 %`, I believe this should read `0.5%`.

Reply:
`over a target of 40960` might be better as `over target (40,960)` (or something without "a").

Reply:
I would personally prefer to drop the value altogether. Just `x% over/under target`. The value of the target can be found in the appropriate place somebody is interested.